src/Kros.AspNetCore/Authorization/GatewayAuthorizationMiddleware.cs

```diff
@@ -138,6 +142,18 @@ internal class GatewayAuthorizationMiddleware
             }
         }
 
+        private void AddForwardedHeaders(HttpClient client, IHeaderDictionary headers)
+        {
+            StringValues value;
+            foreach(string headerName in _jwtAuthorizationOptions.ForwardedHeaders)
+            {
+                if(headers.TryGetValue(headerName, out value))
+                {
+                    client.DefaultRequestHeaders.Add(headerName, value.ToString());
```

Comment:
Nie je nutné volať `ToString()` a zbytočne tak vyťvárať reťazec. Stačí to pretypovať: `IEnumerable<string>)value`.